Makefile

```diff
@@ -154,12 +148,11 @@ mypy: ## Type checking - mypy
 schema: ## Dumps graphql schema in schema.json
 	$(COMMAND_PREFIX) ./manage.py graphql_schema --schema uobtheatre.schema.schema --out schema.graphql
 
-pr: ## Runs everything required (that is not included in precommit) for a pr
+pr: ## Runs everything required for a pr
```

Comment:
Agree precommit is annoying and we should remove it but we should add back the functionality it provided here. It did all sorts.but most importantly lint and mypy

Reply:
The functionality is already included in make lint?